ui/job-view/redux/stores/selectedJob.js

```diff
@@ -240,7 +261,7 @@ export const changeJob = (
       if (jobInstance) {
         // Delay updating details for the new job right away,
         // in case the user is switching rapidly between jobs
-        return doSelectJob(jobInstance.props.job, false);
```

Comment:
I'm curious how this will respond on prototype.  It's sluggish when switching between jobs using `n` and `p` as well as arrow right and left on my machine.  But the DB queries are much slower.  So it may just be that.  Passing in `updateJobDetails` here was an optimization.  I suppose we will find out when we get the Sheriffs to test-drive this before pushing to prod.  :)